src/main.c

```diff
@@ -1879,15 +1879,15 @@ int main(int argc, char **argv)
     glBindTexture(GL_TEXTURE_2D, texture);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
-    load_png_texture("textures/texture.png");
+    load_png_texture("../textures/texture.png");
 
     GLuint font;
     glGenTextures(1, &font);
     glActiveTexture(GL_TEXTURE1);
     glBindTexture(GL_TEXTURE_2D, font);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
-    load_png_texture("textures/font.png");
+    load_png_texture("../textures/font.png");
```

Comment:
If I revert the previous path, then the error gets thrown here (and the same would happens for the paths below).

Reply:
Weird. So for you, not having "../" before the files works and breaks when you add them?

Reply:
Yeah. I'm not sure why though. The path you had makes sense to me. Maybe there is a way to resolve the absolute path or something.

Reply:
I changed it to use the absolute path, so hopefully that will work. It's not the best way to do the pathing, but I'm really not sure why it isn't working.